YarnSpinner/Value.cs

```diff
@@ -0,0 +1,351 @@
+
+using System;
+
+namespace Yarn
+{
+	// A value from inside Yarn.
+	public class Value : IComparable, IComparable<Value> {
+		public static readonly Value DEFAULT = new Value();
```

Comment:
This feels a bit weird. It looks like the `DEFAULT` value is only used for the case where a variable cannot be found, and is always null; if that's the case, why isn't it called null? And if that change was made, shouldn't all null values simply be references to this instance?